oas_docs/output/kibana.serverless.yaml

```diff
@@ -13356,6 +13356,185 @@ paths:
       summary: Health check on Privilege Monitoring
       tags:
         - Security Entity Analytics API
+  /api/entity_analytics/monitoring/users:
+    post:
+      operationId: CreatePrivMonUser
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                is_monitored:
```

Comment:
All our other APIs seem to _mostly_ use lowerCamelCase, I don't think we should deviate from that if we can help it. What do you think?

Reply:
and regarding is_monitored, what is the advantage of transforming the data from the ECS format? e.g I believe this should be `labels.monitoring.privileged_users`